packages/grafana-ui/src/components/Layout/Space.mdx

```diff
@@ -0,0 +1,22 @@
+import { Meta, ArgTypes } from '@storybook/blocks';
+import { Space } from './Space';
+
+<Meta title="MDX|Space" component={Space} />
+
+# Space
```

Comment:
I'm wondering if we should have something here (or in Saga) explaining what the `layout` prop is for? Like when should you use `inline` or `block`?